nixos/modules/module-list.nix

```
@@ -1163,7 +1163,12 @@
   ./system/boot/stage-1.nix
   ./system/boot/stage-2.nix
   ./system/boot/systemd.nix
+  ./system/boot/systemd-coredump.nix
```

```suggestion
  ./system/boot/systemd/coredump.nix
```

Imo would be nicer like this

---

I agree -- if we're splitting systemd into related modules, they should all go under an umbrella `systemd` directory.

---

I like this one as well. My previous approach aligned with `systemd-nspawn`. I have moved `systemd-nspawn` to the `systemd` directory as well in a separate commit. :+1: